### PR TITLE
fix #731:Toolbar text getting changed on orientation change

### LINKED ISSUE #731: Toolbar label getting changed and minimized on orientation changes
**Summary:**

Toolbar label is modified when the orientation of the screen changes.

**Steps to reproduce:**

1) Run the app
2) Select charges from navbar
3) Change orientatation.
4) Toolbar text changes and is minimized

**Expected behaviour:**

No change is fragment name.

**Observed behaviour:**

Fragment name gets modified.

**Device and Android version:**

Android 5.0.2,API 21

**Screenshots:**

I would like to work on this.

### PR DESCRIPTION
**Pull request by AkshGautam**
Fixes #731 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@therajanmaurya @dilpreet96 Please review

File: app/src/main/res/values/styles_toolbar.xml [hunk 1]
```diff
@@ -18,4 +18,7 @@
         <item name="android:windowActionBarOverlay">false</item>
     </style>
 
+    <style name="ToolbarTitle" parent="@style/TextAppearance.AppCompat.Title">
+        <item name="android:textSize">20sp</item>
```
Comment (therajanmaurya) [inline app/src/main/res/values/styles_toolbar.xml:22]: why size 20dp. I don't find material guideline.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/styles_toolbar.xml (hunk 1 above)
Comment (AkshGautam) [reply app/src/main/res/values/styles_toolbar.xml]: What should be the size then?

Comment (AkshGautam) [reply app/src/main/res/values/styles_toolbar.xml:22]: @therajanmaurya I found out that the default size for the portrait mode text is 20sp while it shrinks down to 14sp for landscape mode. I have tried to make the size standard by fixing it to 20sp i.e similar to that of portrait mode. What changes should I make?

Comment (AkshGautam) [reply app/src/main/res/values/styles_toolbar.xml:22]: @therajanmaurya Here is the guideline [link](https://material.io/guidelines/style/typography.html#) which says App bar: Title style, Medium 20sp

**Review by therajanmaurya — APPROVED** [review]

File: app/src/main/res/values/styles_toolbar.xml (hunk 1 above)
Comment (therajanmaurya) [reply app/src/main/res/values/styles_toolbar.xml:22]: ok

**Pull request merged**